code/index.html

```diff
@@ -4,27 +4,200 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="style.css">
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Bebas+Neue&display=swap');
+     </style>
   </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
+ 
+<body>
+   
+    
+    <!-- Navbar-->
+   
+    <nav>
+        <div class="nav-content">
+        <div class="nav-logo"><img src="images/whiteicon.png"></div>
+        
+        <div class="nav-links">
+          <a href="#" class="nav-link">About</a>
+          <a href="#" class="nav-link">Coach</a>
+          <a href="#" class="nav-link">Log in</a>
+          <a href="#" class="nav-link">Member</a>
+          <a href="#" class="nav-link">Blog</a>
+        </div>
+    </nav>
 
-    <!-- Signup form -->
+    <!-- Video Header -->
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+    <header>  
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <section class="header-content">
+        <img src="images/whiteicon.png" class="logo">
+        <p>Are you our new coach?</p>
+
+          <section class="header-content">
+          <button class="button-header"><p>Scroll Down</p></button>
+          </section>
+
+      </section>
+
+      <div class="video-container">
+          <video id= "videoBG" poster="images/poster.png" autoplay muted loop>
+            <source src="videos/freemovr.mp4" type="video/mp4"/>
+          </video>
+      </div>
+
+    </header>
+
+
+             <!-- Signup form -->
```

Comment:
Very helpful with these comments! It is easy to read and find the right section :)

Reply:
Thank you!